modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java

```diff
@@ -106,6 +108,11 @@ public Map<String, Object> getParameters(
 		).put(
 			"options",
 			() -> {
+				if (objectFieldId > 0) {
```

Comment:
Only one use, please inline it.

Reply:
The SF asks for variable:
 
`[java] Create a new variable for the left hand side operand of the ">" operator for better readability: ./modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java 100 (Checkstyle:OperatorOperandCheck)
`